fighthealthinsurance/static/js/blog_post.tsx

```
@@ -1,25 +1,150 @@
+renderer.heading = (text: string, level: number, raw: string) => {
+  // Check for explicit ID in markdown like {#custom-id}
+  const explicitIdMatch = raw.match(/(.+?)\s*\{#([\w-]+)\}$/);
+  let slug: string;
+  let headingText: string = text;
+  if (explicitIdMatch) {
+    headingText = explicitIdMatch[1].trim();
+    slug = explicitIdMatch[2];
+  } else {
+    slug = raw
+      .toLowerCase()
+      .replace(/[^\u0000-\u007F\w\s-]/g, '') // remove non-ascii
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)